lib/Dialect/Tpu/Interfaces/BM1684X/Store.cpp

```diff
@@ -41,12 +41,12 @@ void tpu::StoreOp::codegen_local_bm1684x(int64_t n_step, int64_t h_step,
     gdma_format = BM168x::GDMA_VALUE_FORMAT_INT8;
     data_type = DTYPE_INT8;
     if (gi.h_slice == H) {
-      if (W * H & 0x1 == 1) {
```

Comment:
Very intresting that it is evaluated as `(W *H) & (0x1 == 1)` but the result is same as `(W * H & 0x1) == 1`.